src/ClaimService.Business/Features/Claims/Commands/Edit/EditClaimValidator.cs

```diff
@@ -82,13 +87,29 @@ public class EditClaimValidator : BaseEditRequestValidator<EditClaimRequest>, IE
       {
         {
           async (x) => x.value is not null && !Guid.TryParse(x.value.ToString().Trim(), out Guid categoryId) &&
-            await _provider.Categories.AnyAsync(c => c.Id == categoryId && c.IsActive),
+            !await _provider.Categories.AnyAsync(c => c.Id == categoryId && c.IsActive),
           "Incorrect category id value."
         }
       });
 
     #endregion
 
+    #region DepartmentId
+
+    await AddFailureForPropertyIfAsync(
+      nameof(EditClaimRequest.DepartmentId),
+      x => x == OperationType.Replace,
+      new()
+      {
+        {
+          async (x) => x.value is not null && !Guid.TryParse(x.value.ToString().Trim(), out Guid departmentId) &&
```

Comment:
it seems that negation operator is redundant and on 106 line is too

Reply:
agree about 105, but 106 is required for sure. same with 89-90